src/main/java/co/inventorsoft/academy/collections/model/Range.java

```diff
@@ -1,60 +1,126 @@
 package co.inventorsoft.academy.collections.model;
 
+
+import java.util.ArrayList;
 import java.util.Collection;
 import java.util.Iterator;
+import java.util.List;
 import java.util.Set;
+import java.util.function.Function;
 
 public class Range<T> implements Set<T> {
 
+    @Override
+    public String toString() {
+        return "Range{" +
+                "expectedElements=" + expectedElements +
+                '}';
+    }
+
+    public final List<T> expectedElements = new ArrayList<>();
+
+    public static class IntegerNextValue implements Function<Integer, Integer> {
+
+        @Override
+        public Integer apply(Integer i) {
+            return i + 1;
+        }
+    }
+
+
+    public static Range<Integer> of(Integer first, Integer last) {
+        IntegerNextValue inv = new IntegerNextValue();
+        Range<Integer> rangeOf = Range.of(first, last, inv);
+
+        return rangeOf;
+    }
+
+    public static class FloatNextValue implements Function<Float, Float> {
+        @Override
+        public Float apply(Float f) {
+            return f + 0.1f;
+        }
+    }
+
+    public static Range<Float> of(Float first, Float last) {
+        Function<Float, Float> fnv = new FloatNextValue();
+        Range<Float> floatRange = Range.of(first, last, fnv);
```

Comment:
You don't use this variable, return value immediately

Reply:
ok